### DISCUSSION TITLE
É possivel deletar repositorio criado?

### DISCUSSION THREAD

**Discussion by danielaz87** [community/community#61955] [New to GitHub]
Bom dia!

É possível deletar repositorio criado?
Fui criar e fiz errado e não consegui deletar, alguém sabe se é possivel?

**Comment by queenofcorgis** [comment]
Hi there - our docs should help you out with that, please review [this one](https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository) for instructions.

**Comment by LeandroLuna** [comment]
Olá, Daniela!

Sim, é possível deletar um repositório. 
Para tal, siga os passos abaixo:

1. Acesse o Repositório: Vá para a página principal do seu perfil ou da organização onde o repositório está localizado. Em seguida, clique no nome do repositório que deseja excluir para acessar a página do repositório.

2. Acesse as Configurações: Na página do repositório, clique na aba "Settings" (Configurações) localizada na parte superior direita da página.

3. Role a Página para Baixo: Na página de configurações do repositório, role para baixo até encontrar a seção "Danger Zone".

4. Excluir o Repositório: Dentro da seção "Danger Zone", você encontrará a opção para "Delete this repository" (Excluir este repositório). Clique nesse link.

Espero que tenha ajudado!
